.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,47 +1,37 @@
-name: Bug Report
-description: Create a bug report to help improve FluentHub
+name: 🐛 Bug Report
+description: Create a bug report to help us improve FluentHub!
 labels: [bug]
-assignees: [onein528]
+assignees: [0x5bfa]
+title: "Bug: "
 body:
-  - type: checkboxes
-    attributes:
-      label: Before you start
-      options:
-        - label: Have you updated FluentHub? You might be using an old version.
-          required: true
-        - label: Have you checked if a similar issue has already been reported?
-          required: true
   - type: dropdown
     id: windows
     attributes:
-      label: What version of Windows is this issue present?
+      label: 🪟 What version of Windows is this issue present?
       multiple: true
       options:
-        - Windows 10
-        - Windows 10 Release Preview
-        - Windows 10 Beta
-        - Windows 10 Dev
-        - Windows 11
-        - Windows 11 Release Preview
-        - Windows 11 Beta
-        - Windows 11 Dev
+        - 🪟 Windows 10
+        - 🪟 Windows 10 Release Preview
+        - 🪟 Windows 10 Beta
+        - 🪟 Windows 10 Dev
+        - 🪟 Windows 11
+        - 🪟 Windows 11 Release Preview
+        - 🪟 Windows 11 Beta
+        - 🪟 Windows 11 Dev
     validations:
       required: true
   - type: textarea
     attributes:
-      label: What version/architecture of FluentHub are you on?
-      description: Copy and paste the version from the about page of FluentHub's settings. If you're building from GitHub, simply provide the commit hash you built on.
-    validations:
-      required: true
```

Comment:
Why description and validation was removed?

Reply:
My mistake - reverting now.